src/main/java/org/corfudb/infrastructure/SequencerServer.java

```diff
@@ -143,17 +145,19 @@ public synchronized void handleMessage(CorfuMsg msg, ChannelHandlerContext ctx,
                 if (req.getNumTokens() == 0) {
                     long max = 0L;
                     boolean hit = false;
-                    ImmutableMap.Builder<UUID, Long> streamsLastIssued = ImmutableMap.builder();
+                    ImmutableMap.Builder<UUID, Long> latestStreamOffsets = ImmutableMap.builder();
```

Comment:
'max' seems to be the maximum value of global-offsets for all streams in this requests, and if no streams are specified, the last issued global offset. Correct? Please comment, and change the name to something meaningful